include/igl/opengl/ViewerData.h

```diff
@@ -250,6 +250,7 @@ class ViewerData
   unsigned int show_overlay;
   unsigned int show_overlay_depth;
   unsigned int show_texture;
+  unsigned int double_sided;
```

Comment:
I'd say it should be a `bool` right below the `face_based` toggle. Or do we want to make this a per-viewport property?

Reply:
sure go ahead.